sample/android/src/main/java/co/nimblehq/kmm/template/ui/base/BaseViewModel.kt

```diff
@@ -42,11 +43,12 @@ abstract class BaseViewModel : ViewModel() {
         }
     }
 
+    protected fun launch(context: CoroutineContext = EmptyCoroutineContext, job: suspend () -> Unit) =
+        viewModelScope.launch(context) {
+            job.invoke()
```

Comment:
Optional:
```suggestion
            job()
```

Reply:
@kaungkhantsoe This is auto-generated from the implementation in android-templates > template-compose. It will be updated automatically after fixing the issue from there 💁‍♂️ We can skip it here.